.github/workflows/sanity_test.yml

```diff
@@ -78,3 +78,17 @@ jobs:
           path: |
             emulator.log
             failure_screenshots/
+
+
+  notify:
```

Comment:
what happens if the matrix secret is empty? thinking of what will happen for forks~ (if they have actions enabled)

Reply:
I haven't tested this explicitly, though I have tested by setting the secrets to invalid values.